src/styles/app/05-main-area.css

```
@@ -468,148 +183,71 @@
-	font-weight: var(--font-medium);
-	color: var(--text-primary);
+	font-weight: var(--font-semibold);
 }
 
-.welcomeReadyRecentPath {
-	font-size: 11px;
+.welcomeRecentPath {
+	font-size: var(--text-xs);
 	color: var(--text-tertiary);
-	word-break: break-all;
+	word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)